section4/exercises/classMethods.js

```diff
@@ -28,13 +28,28 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
 
-  // ADD CODE
+  changeProtein(newProtein) {
+    this.protein = newProtein;
+  }
+  addTopping(newTopping) {
+    this.toppings.push(newTopping);
+  }
 };
 
-// ADD CODE
+var burritoSalad = new Burrito('Chicken', 'Lettuce', ['Pico de Gallo, Dressing, Avocado']);
```

Comment:
Careful of your syntax for the `toppings` arguments. I suspect this is more of a typo because you have this right in other areas of the prework, but a quick refresher:

In each of the below lines where you instantiate a Burrito, you pass in a single string instead of a list of strings.

`['Pico de Gallo, Dressing, Avocado']`

^so there's just one element in this array (a big string).

`['Pico de Gallo', 'Dressing', 'Avocado']`

^and there's 3 string elements in this array.